ksqldb-rest-app/src/test/java/io/confluent/ksql/rest/integration/HighAvailabilityTestUtil.java

```diff
@@ -181,7 +182,10 @@ public static void sendLagReportingRequest(
           .exceptionally(t -> {
             LOG.error("Unexpected exception in async request", t);
             return null;
-          });
+          })
+      .get();
+    } catch (Exception e) {
+      LOG.error("Error waiting on lag report", e);
```

Comment:
Catching and swallowing in a test util...  are we sure this is the right approach?

Reply:
You're right, I should let it bubble up and fail immediately (as opposed to letting a following assertion fail).  The important part was calling `get` since the async aspect was causing tests to have some flakiness.